libraries/AP_NavEKF3/AP_NavEKF3_AirDataFusion.cpp

```diff
@@ -447,6 +452,318 @@ void NavEKF3_core::FuseSideslip()
     ConstrainVariances();
 }
 
+/*
```

Comment:
ifdef for features

Reply:
??

Reply:
sorry Paul, this was a comment to myself to think about adding ifdefs in the EKF code for features like this